misk/src/test/kotlin/misk/client/ClientMetricsInterceptorTest.kt

```diff
@@ -40,17 +41,19 @@ internal class ClientMetricsInterceptorTest {
   @MiskTestModule
   val module = TestModule()
 
-  @Inject private lateinit var jetty: JettyService
+  @Inject
+  private lateinit var jetty: JettyService
   private lateinit var requestDuration: Histogram
   private lateinit var clientMetrics: Metrics
   private lateinit var clientInjector: Injector
 
   @BeforeEach
   fun createClient() {
-    clientInjector = Guice.createInjector(ClientModule(jetty))
+    clientInjector = Guice.createInjector(ClientModule(jetty)) //this should be changed to the approach of MetricsInterceptorTest.kt?
```

Comment:
probably not. this test is trying to simulate the difference between a client and a server running into 2 different apps, which would result in 2 different injectors (1 injector per app)

Reply:
Thanks for clarifying